app/views/evaluators/_delete_evaluator_modal.html.erb

```diff
@@ -34,15 +34,5 @@
         </ul>
       </div>
     </div>
-    <button
```

Comment:
there's no close icon in Figma and this markup was erroring out because the sprite.svg icon was not found so I just removed it here